src/analytics/SegmentProvider.test.tsx

```
@@ -0,0 +1,159 @@
+        <ChromeAuthContext.Provider value={mockAuthValue}>
+          <SegmentProvider>
+            <div>Test Child</div>
+          </SegmentProvider>
+        </ChromeAuthContext.Provider>
+      );
+    }).not.toThrow();
+
+    // Verify defensive warning was logged
+    expect(console.warn).toHaveBeenCalledWith(
+      'Router context not initialized in SegmentProvider, using default location'
+    );
```

ESLint

[error] Replace `⏎······'Router·context·not·initialized·in·SegmentProvider,·using·default·location'⏎····` with `'Router·context·not·initialized·in·SegmentProvider,·using·default·location'`

(prettier/prettier)